integrationtests/accountsESDTRollback_test.go

```diff
@@ -26,7 +26,7 @@ func TestAccountsESDTDeleteOnRollback(t *testing.T) {
 
 	esdtToken := &esdt.ESDigitalToken{
 		Value:      big.NewInt(1000),
-		Properties: []byte("ok"),
+		Properties: []byte("3032"),
```

Comment:
why not testing with `6f6b` **as the original test was**? 
In fact this is sent by the node as `hex("ok")`